### ISSUE TITLE
update db from day-end-form and nrcan

### ISSUE THREAD

**Issue by tevpg** [ironwoodcall/TagTracker#239]
Automatic merging of gsheet day end into db.  Also env canada (nrcan) weather data.

script will assume that data is present in local file system in csv format.

getting gsheet in csv:
```
GSheetURL='https://docs.google.com/spreadsheets/d/{google-id}/gviz/tq?tqx=out:csv&gid={sheet-id}'
curl $GSheetURL -o whatever.csv
```

Values to likely pull from external sources, all in DAY table:
- registrations
- reported-abandoned
- precip
- temp

Tricky bit:
- tracker2db.py will need to only UPDATE and INSERT its DAY records as it no longer owns the whole rows.

How I want it to work ... thinking ahead of use for similar job of NRCan data:
- ideally. parameters:
  - CSV of data to insert
  - DATE and VALUE columns in CSV
  - database name
  - database table:column name
  - indication whether all or missing or DATE
- simpler.  :star: Specific front-ends tuned to NRCan or to day-end; so now parms are:
  - CSV file location
  - database location
  - all, empty, DATE
- possible compromise:
  - key e.g. 'day-end' or 'weather'
  - CSV file location
  - database location
  - all, empty, DATE 
- common back-end dbutil can update table:column in open db file. parameters:
  - key to match (eg date)
  - value to insert
  - table, key column, target column
  - basically this would `UPDATE {table} SET {targetcol} = {targetval} WHERE {keycol} = {keyval}` with lots of error checking
  - this does mean that if doing a whole whack of updates it will be a whole whack of calls, but (1) there are only a few hundred rows at worst, and (2) generally it will be a small number of updates (e.g. one row per day for each value type, of which I expect maybe three or four)
